packages/hdwallet-gridplus/package.json

```diff
@@ -11,15 +11,16 @@
   "scripts": {
     "build": "tsc --build",
     "build:docs": "typedoc --out docs --target es6 --theme minimal --mode file src",
-    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "clean": "rm -rf dist node_modules tsconfig.tsbuildinfo",
```

Comment:
This currently breaks publish, see other review comment

Reply:
Ok, I will get this fixed. We want clean to actually clean all artifacts, so the prepublishOnly script needs to be updated to only clean build artifacts before publish.